Sources/Sentry/SentryAutoSessionTrackingIntegration.m

```diff
@@ -0,0 +1,37 @@
+#if __has_include(<Sentry/Sentry.h>)
+#import <Sentry/SentryAutoSessionTrackingIntegration.h>
+#import <Sentry/SentrySessionTracker.h>
+#import <Sentry/SentryOptions.h>
+#import <Sentry/SentryHub.h>
+#import <Sentry/SentrySDK.h>
+#import <Sentry/SentryLog.h>
+#else
+#import "SentryAutoSessionTrackingIntegration.h"
+#import "SentrySessionTracker.h"
+#import "SentryOptions.h"
+#import "SentryHub.h"
+#import "SentrySDK.h"
+#import "SentryLog.h"
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryAutoSessionTrackingIntegration()
```

Comment:
it's called `SessionTrackingIntegration` on Android, should we match names? I don't mind changing it as well.

Reply:
I named it like this to match the other integration in this SDK